itests/testdata/transfer_tx_testdata.go

```diff
@@ -49,38 +71,389 @@ type CommonTransferData struct {
+				WavesDiffBalanceRecipient: wavesAmount / 8,
+			}),
+		"Transfer assets to oneself": *NewTransferTestData(
+			utl.GetAccount(suite, utl.DefaultSenderNotMiner),
+			proto.NewRecipientFromAddress(utl.GetAccount(suite, utl.DefaultSenderNotMiner).Address),
+			&assetId,
+			nil,
+			utl.MinTxFeeWaves,
+			uint64(assetAmount/8),
+			utl.GetCurrentTimestampInMs(),
+			utl.TestChainID,
+			proto.Attachment("2GjX8YcCcSdmYm3pVP41e1TL1t5nQrHUkCx6V9L7SC5LxQSEmcE3irKh2NtV2x57fNU5MoRM"),
+			TransferExpectedValuesPositive{
+				WavesDiffBalanceSender:    utl.MinTxFeeWaves,
+				AssetDiffBalance:          0,
+				WavesDiffBalanceRecipient: utl.MinTxFeeWaves,
+			}),
+		"Transfer waves to oneself": *NewTransferTestData(
+			utl.GetAccount(suite, utl.DefaultSenderNotMiner),
+			proto.NewRecipientFromAddress(utl.GetAccount(suite, utl.DefaultSenderNotMiner).Address),
+			nil,
+			nil,
+			utl.MinTxFeeWaves,
+			uint64(wavesAmount/8),
+			utl.GetCurrentTimestampInMs(),
+			utl.TestChainID,
+			proto.Attachment("2GjX8YcCcSdmYm3pVP41e1TL1t5nQrHUkCx6V9L7SC5LxQSEmcE3irKh2NtV2x57fNU5MoRM"),
+			TransferExpectedValuesPositive{
+				WavesDiffBalanceSender:    utl.MinTxFeeWaves,
+				AssetDiffBalance:          0,
+				WavesDiffBalanceRecipient: utl.MinTxFeeWaves,
+			}),
+		"Address as string": *NewTransferTestData(
+			utl.GetAccount(suite, utl.DefaultSenderNotMiner),
+			rcpntAddress,
+			&assetId,
+			nil,
+			utl.MinTxFeeWaves,
+			uint64(assetAmount/8),
+			utl.GetCurrentTimestampInMs(),
+			utl.TestChainID,
+			nil,
+			TransferExpectedValuesPositive{
+				WavesDiffBalanceSender:    utl.MinTxFeeWaves,
+				AssetDiffBalance:          assetAmount / 8,
+				WavesDiffBalanceRecipient: 0,
+			},
+		),
+		"Alias as string": *NewTransferTestData(
+			utl.GetAccount(suite, utl.DefaultSenderNotMiner),
+			rcpntAlias,
+			nil,
+			nil,
+			utl.MinTxFeeWaves,
+			uint64(wavesAmount/8),
+			utl.GetCurrentTimestampInMs(),
+			utl.TestChainID,
 			nil,
 			TransferExpectedValuesPositive{
-				WavesDiffBalance: 100000,
-				AssetDiffBalance: 1,
+				WavesDiffBalanceSender:    utl.MinTxFeeWaves + wavesAmount/8,
+				AssetDiffBalance:          0,
+				WavesDiffBalanceRecipient: wavesAmount / 8,
 			},
 		),
 	}
+	return t
+}
+
+func GetTransferMaxAmountPositive(suite *f.BaseSuite, assetId crypto.Digest, accNumber int) map[string]TransferTestData[TransferExpectedValuesPositive] {
+	wavesAmount := utl.GetAvailableBalanceInWavesGo(suite, utl.GetAccount(suite, accNumber).Address)
+	var t = map[string]TransferTestData[TransferExpectedValuesPositive]{
+		"Max values for amount, attachment": *NewTransferTestData(
+			utl.GetAccount(suite, accNumber),
+			proto.NewRecipientFromAddress(utl.GetAccount(suite, utl.DefaultRecipientNotMiner).Address),
+			&assetId,
+			nil,
+			uint64(wavesAmount),
+			utl.MaxAmount,
+			utl.GetCurrentTimestampInMs(),
+			utl.TestChainID,
+			proto.Attachment("2GjX8YcCcSdmYm3pVP41e1TL1t5nQrHUkCx6V9L7SC5LxQSEmcE3irKh2NtV2x57fNU5MoRML6CVyKVbatfbcNWst"+
```

Comment:
Make it smth like - "This is 140 bytes string....", otherwise it seems, that tis is base58 encoded stating, but it's not)

Reply:
Fix